### PR TITLE
README & Other Doc Updates

### PR DESCRIPTION
**Pull request by shannona**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 ![](images/logos/gordian-signer-screen.jpg)
 
-**Gordian Cosigner** allows users to participate in a multisig by adding a signature to an otherwise unsigned or partially signed PSBT, where they may or may not have initiated the transaction. It can also be used for signing a single-signature PSBT. It's largely intended as an offline signing tool, which allows signing without the usage of a full node: a real wallet is need to both initiate an account map and to initiate a transaction.
+**Gordian Cosigner** allows users to participate in a multisig by adding a signature to an otherwise unsigned or partially signed PSBT that was created on another device. It's largely intended as an offline signing tool, which allows signing without a direct connection to a full node: a real wallet is need to initiate a transaction.
```
Comment (ChristopherA) [inline README.md]: Can we use some other term than "real wallet"? Wallets are apps, and this is an app, and thus a wallet. What is needed is either wallet that can initiate a PSBT transaction, or transaction coordinator service.

Comment (shannona) [reply README.md]: Yep. Hadn't even meant to PR this yet.

File: README.md [hunk 2]
```diff
@@ -23,20 +25,35 @@ GordianCosigner-Catalyst is currently a late alpha. It should not be used for pr
...
 
-## Installation
+## Installation Instructions
 
 **Gordian Cosigner** is available for testing from Testflight [here](https://testflight.apple.com/join/sJTaoUsM).
 
-## Usage
+## Usage Instructions
+
+**Gordian Cosigner** supports the middle step of multisig signing, after a multisig has been Initiated, while it is being Cosigned, and before it is Finalized. Usually, it will be used by a cosigner holding one of the keys for a multisignature before handing it back to another user for finalization.
```
Comment (ChristopherA) [inline README.md]: "one of the keys for a multisignature before" — how about "one of the keys used in a multisig account transaction, before".

File: README.md [hunk 3]
```diff
@@ -23,20 +25,35 @@ GordianCosigner-Catalyst is currently a late alpha. It should not be used for pr
...
+
+### Preparing for Usage
+
+To prepare **Gordian Cosigner** for usage, you must first create a copy of the multisig that is being used. This is done by defining an account containing all of the cosigners in the multisig. The user of **Gordian Cosigner** will typically have a private key or seed for his own cosigner element, and then will typically be given xpubs or QR codes for other users' cosigner elements. When he combines these into an account, it should match the multisig created by other people on their own wallets. (See our [Scenarios page](Docs/Scenarios.md) for more discussion of how **Gordian Cosigner** might be used.)
```
Comment (ChristopherA) [inline README.md]: Typically the GCS user receives an account from the initiator, like I sent you a 2 of 2 today. The ability to create your own accounts in GCS is a convenience, mostly for ease of testing, not a general practice.

File: README.md (hunk 1 above)
Comment (shannona) [reply README.md]: (Done.)

File: README.md (hunk 2 above)
Comment (shannona) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (shannona) [reply README.md]: I've revised the discussion of the "create" testing methods.

**Review by ChristopherA — APPROVED** [review]
ACK — read through and text changes all reasonable.

**Pull request merged**